pkg/db/db.go

```diff
@@ -433,6 +438,7 @@ func (g *GormDB) Watch(ctx context.Context, criteria WatchCriteria) (chan Record
 				}()
 				return
 			case rec, ok := <-merged:
+				_ = g.decryptData(ctx, &rec)
```

Comment:
I don't like that I had to do this. There isn't a clean way to handle any kind of decryption error here, so we just ignore it. I'm also not even sure why the records that arrive here are ever encrypted in the first place, since I thought that Watch would call other functions that already return decrypted records. But without this line, the watches do not work, so idk.

Reply:
This should probably be moved to since() method.  I think that is the proper place and error handling should be better.  If decryption fails, it will just break all watches.  Which is really the only thing you can do.